src/main/frontend/src/components/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import {useNavigate} from "react-router-dom";
+import {Button, Form, Input, message} from "antd";
+import React, {useState} from "react";
+import axios, {AxiosResponseHeaders} from "axios";
+import {login, Tokens} from "../slices/authSlice";
+import {useDispatch} from "react-redux";
+import {GithubOutlined, GoogleOutlined} from "@ant-design/icons";
+import {AppDispatch} from "../store";
+
+interface LoginData {
+    loginId: string,
+    password: string
+}
+
+function Login(): JSX.Element {
+    const navigate = useNavigate();
+    const dispatch = useDispatch<AppDispatch>();
+    const [loginData, setLoginData] = useState<LoginData>({loginId: "", password: ""});
+
+    const popupWindowFeature:string = 'top=30,left=30,width=700,height=600,status=no,menubar=no,toolbar=no,resizable=no';
+
+    const saveTokens = (headers: AxiosResponseHeaders) => {
+        const tokens:Tokens = {accessToken : headers['authorization'], refreshToken : headers['refresh-token']};
+        dispatch(login(tokens));
+        navigate('/');
+    };
+
+    const openOauthPopup = (registrationId:string) => {
+        window.open('/oauth2/authorization/' + registrationId, registrationId + 'Window', popupWindowFeature);
+    };
+
+    const onChange = (e:React.ChangeEvent<HTMLInputElement>) => {
+        const {name, value} = e.target;
+        setLoginData({...loginData, [name]: value});
+    };
+
+    const onFinish = () => {
+        axios.post('/api/login', loginData)
+            .then((result) => {
+                saveTokens(result.headers)
+            })
+            .catch((error) => {
+                message.error(error.response.data.message);
+            })
+    };
+
+    return (
+        <div>
+            <div className="title">모두의 모임</div>
+            <Form
+                name="basic"
+                layout='vertical'
+                labelCol={{offset: 8}}
+                wrapperCol={{offset: 8, span: 8}}
+                onFinish={onFinish}
+                autoComplete="on"
+                size={"large"}
+            >
+                <Form.Item label="Login ID" required>
+                    <Input name={"loginId"} placeholder="input Login ID" required={true}
+                           onChange={(e) => {onChange(e)}}/>
```

Comment:
이렇게도 축약이 가능합니다. 자바의 메서드참조랑 동일한 개념이라고 생각하면 될거에요
```suggestion
                    <Input name={"loginId"} placeholder="input Login ID" required={true}
                           onChange={onChange}/>
```